executor/window_test.go

```diff
@@ -311,8 +312,15 @@ func (s *testSuite7) TestSlidingWindowFunctionsSumDecimal(c *C) {
 	tk.MustExec("use test;")
 	tk.MustExec("drop table if exists t;")
 	tk.MustExec("CREATE TABLE t (id DECIMAL, sex CHAR(1));")
-	for i := 0; i < 100000; i++ {
-		tk.MustExec(fmt.Sprintf("insert into t values (%d,'M')", i%10))
+	var builder strings.Builder
```

Comment:
After communicating with @mmyj, `TestSlidingWindowFunctionsSumDecimal` is useless now and can be removed.